src/devtools/DevtoolsGraphPanel.js

```diff
@@ -14,12 +14,7 @@ export class DevtoolsGraphPanel {
    * @param {Utils.Observer<Audion.GraphContext>} graphObserver
    */
   constructor(graphObserver) {
-    chrome.devtools.panels.create(
-      'Web Audio',
-      '',
-      chrome.runtime.getUrl('panel.html'),
-      () => {},
-    );
+    chrome.devtools.panels.create('Web Audio', '', 'panel.html', () => {});
```

Comment:
Perhaps `Web Audio Inspector` might be better.

Reply:
Ah. I'm not sure that makes sense. This `'Web Audio'` string appears in the bar of panels at the top of devtools. I think `Inspector` would be redundant.

![image](https://user-images.githubusercontent.com/833298/128052353-b36b3b4a-a676-4afb-8904-389d234c75bb.png)

Reply:
We can discuss this later and don't have to settle on the name right now.

But we already have "Web Audio" in the devtools, so definitely need something different:
https://web.dev/profiling-web-audio-apps-in-chrome/#use-the-webaudio-tab-in-chrome-devtools

Reply:
I will resolve this for now, so we can move one.